lambdas/service/app.py

```diff
@@ -650,6 +652,18 @@ def validate_filters(filters):
             else:
                 raise BadRequestError(f'The relation in the `filters` parameter entry for `{facet}`'
                                       f' must be one of {valid_relations}')
+            if facet == 'organismAge':
+                for f in value:
+                    try:
+                        reject('' in (f['value'], f['unit']),
+                               f'Facet {facet} should not specify empty strings values',
+                               exception=BadRequestError)
+                    except KeyError:
```

Comment:
What if `reject` raises a KeyError? You'd be blaming the client for a bug in the server.